### PR TITLE
Changed markups to be loggable

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Services/Markups/IMarkupPolicyService.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using System.Threading.Tasks;
+using HappyTravel.Edo.Api.Models.Agents;
+using HappyTravel.Edo.Common.Enums.Markup;
+using HappyTravel.Edo.Data.Markup;
+
+namespace HappyTravel.Edo.Api.Services.Markups
+{
+    public interface IMarkupPolicyService
+    {
+        Task<List<MarkupPolicy>> GetPolicies(AgentContext agentContext, MarkupPolicyTarget policyTarget);
```
Comment (kirillta) [inline Api/Services/Markups/IMarkupPolicyService.cs:11]: Could be just `Get`

File: Api/Services/Markups/MarkupPolicyService.cs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+            string BuildKey()
+                => _flow.BuildKey(nameof(MarkupPolicyService),
+                    nameof(GetAgentPolicies),
+                    agentId.ToString());
+
+
+            async Task<List<MarkupPolicy>> GetOrderedPolicies()
+            {
+                var policiesFromDb = await GetPoliciesFromDb();
```
Comment (kirillta) [inline Api/Services/Markups/MarkupPolicyService.cs:74]: `GetPoliciesFromDb` uses only once. Could it be merged here to avoid unnecessary iterations?

**Review by kirillta — APPROVED** [review]

File: Api/Services/Markups/MarkupPolicyService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Markups/MarkupPolicyService.cs]: Changed to ordering in db

File: Api/Services/Markups/IMarkupPolicyService.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Services/Markups/IMarkupPolicyService.cs]: Fixed

**Review by DFSko — APPROVED** [review]

**Pull request merged**